test/views/cancellation/CancelDeclarationViewSpec.scala

```diff
@@ -133,11 +132,12 @@ class CancelDeclarationViewSpec extends AppViewSpec with CommonMessages {
 
       val view = createView(CancelDeclaration.form.bind(Map[String, String]()))
 
-      checkErrorsSummary(view)
-      checkErrorLink(view, "functionalReferenceId-error", "error.required", "#functionalReferenceId")
-      checkErrorLink(view, "declarationId-error", "error.required", "#declarationId")
-      checkErrorLink(view, "statementDescription-error", "error.required", "#statementDescription")
-      checkErrorLink(view, "changeReason-error", "error.required", "#changeReason")
+      view must haveGlobalErrorSummary
```

Comment:
We could add separate test section about this. Like
```
should "have error about missing functional reference" in {
  view must haveFieldErrorLink("functionalReferenceId", "#functionalReferenceId")
}
```
Increase granularity and make easier to read failing case from report